queries/wof.jinja2

```diff
@@ -12,7 +12,7 @@ SELECT
       'area', area::bigint,
       'min_zoom', min_zoom,
       'max_zoom', max_zoom,
-      'is_landuse_aoi', CASE WHEN is_landuse_aoi = TRUE THEN TRUE END,
+      'is_landuse_aoi', is_landuse_aoi,
```

Comment:
I'm no following what you mean by moving the min/max zoom to the yaml. Don't we just pass that through here anyway?

I moved the area type conversion to python in ec90ecd.

About the kind for wof records, we just pass that through from the database. But I see where you're coming from, so I hopefully made it clearer by assigning the `kind` name in the yaml off of the field that gets passed through in 1318c55.